README.md

```diff
@@ -19,18 +19,18 @@ To run the **HR Management** Web App, ensure your system meets the following req
 Follow the steps below to setup the project locally:
 
 * clone the repository: 
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone ```https://github.com/harry-olabs/avi-hub.git```
```

Comment:
Like I mentioned before, this line is an entire command code. Wrap it using the format in the documentation.

Spend some time to read the documentation properly use code blocks format (triple backticks on separate lines).